billboardd/dbusbillboard.xml

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+    <interface name="io.thp.billboard">
+        <method name="render">
```

Comment:
yea, i decided on render because its more explicit. refresh could imply that it would re-read the config or properties or something.